sdk/framework/field_data_test.go

```diff
@@ -202,6 +202,105 @@ func TestFieldDataGet(t *testing.T) {
 			0,
 		},
 
```

Comment:
Since 0 is the edge condition, I'd add tests against 0 for both the old and new type.

Reply:
Good idea